synapse/__init__.py

```diff
@@ -20,27 +20,41 @@
 import synapse.lib.modules as s_modules
 
 # load all the synapse builtin modules
-s_modules.load('synapse.models.syn')
-s_modules.load('synapse.models.dns')
-s_modules.load('synapse.models.orgs')
-s_modules.load('synapse.models.inet')
-s_modules.load('synapse.models.mime')
-s_modules.load('synapse.models.files')
-s_modules.load('synapse.models.media')
-s_modules.load('synapse.models.money')
-s_modules.load('synapse.models.telco')
-s_modules.load('synapse.models.crypto')
-s_modules.load('synapse.models.geopol')
-s_modules.load('synapse.models.person')
-s_modules.load('synapse.models.infotech')
-s_modules.load('synapse.models.language')
-s_modules.load('synapse.models.material')
-s_modules.load('synapse.models.temporal')
-s_modules.load('synapse.models.geospace')
-
-s_modules.load('synapse.models.gov.us')
-#s_modules.load('synapse.models.gov.cn')
-s_modules.load('synapse.models.gov.intl')
+# the built-in cortex modules...
+BASE_MODELS = (
+    ('synapse.models.syn.SynMod', {}),
+    ('synapse.models.dns.DnsMod', {}),
+    ('synapse.models.orgs.OuMod', {}),
+    ('synapse.models.inet.InetMod', {}),
+    ('synapse.models.person.PsMod', {}),
+    ('synapse.models.telco.TelMod', {}),
+    ('synapse.models.files.FileMod', {}),
+    ('synapse.models.geopol.PolMod', {}),
+    ('synapse.models.biology.BioMod', {}),
+    ('synapse.models.finance.FinMod', {}),
+    ('synapse.models.infotech.ItMod', {}),
+    ('synapse.models.media.MediaMod', {}),
+    ('synapse.models.money.MoneyMod', {}),
+    ('synapse.models.compsci.CsciMod', {}),
+    ('synapse.models.geospace.GeoMod', {}),
+    ('synapse.models.gov.cn.GovCnMod', {}),
+    ('synapse.models.gov.us.GovUsMod', {}),
+    ('synapse.models.material.MatMod', {}),
+    ('synapse.models.crypto.CryptoMod', {}),
+    ('synapse.models.language.LangMod', {}),
+    ('synapse.models.temporal.TimeMod', {}),
+    ('synapse.models.chemistry.ChemMod', {}),
+    ('synapse.models.science.SciModMod', {}),
+    ('synapse.models.gov.intl.GovIntlMod', {}),
+)
+
+for mod, conf in BASE_MODELS:
```

Comment:
Are you doing these here to facilitate "fail fast"?  I might have thought they'd go best in synapse/cores/common